packages/fluentui/react-northstar/src/components/Video/Video.tsx

```diff
@@ -24,77 +28,86 @@ export interface VideoProps extends UIComponentProps {
 
   /** Video source URL. */
   src?: string;
+
+  accessibility?: Accessibility<VideoBehaviorProps>;
```

Comment:
```suggestion
  // Description ???
  accessibility?: Accessibility<VideoBehaviorProps>;
```

Please copy comment from any other component

Reply:
Done